### DISCUSSION TITLE
Clarifying the new `Load npm builtin configs failed` warning

### DISCUSSION THREAD

**Discussion by stefan-toubia** [pnpm/pnpm#5511] [General]
I'm reviewing the latest release and was hoping to clarify this new warning:

```
 WARN  Load npm builtin configs failed, you can use "pnpm config ls" to show builtin configs. And then use "pnpm config --global set <key> <value>" to migrate configs from builtin to global.
```

I'm really not sure what failed, what are the effects of that failure and what part of my config I'm supposed to move over to the global config.

Also, I did try setting one global config setting per the warning message, but got an additional warning:

```
$ pnpm config --global set cache-dir /Users/stoubia/Library/Caches/pnpm/cache
npm WARN config global `--global`, `--local` are deprecated. Use `--location=global` instead.
```

**Comment by lvqq** [comment]
Related: https://github.com/pnpm/pnpm/issues/5518
